.github/workflows/ci.yml

```
@@ -12,25 +12,25 @@
     steps:
-    - name: Getting configuration from GitHub
-      uses: actions/checkout@v2
+      - name: Getting configuration from GitHub
+        uses: actions/checkout@v2
 
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: ${{ matrix.node-version }}
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
```

actionlint (1.7.4)

the runner of "actions/setup-node@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)